services/node_create.proto

```diff
@@ -27,62 +25,90 @@ option java_multiple_files = true;
- * a-pulbic-NodeAlias.pem file per each node during phase 2,<br>
- * not active until next freeze upgrade.
+ * - A `NodeCreateTransactionBody` MUST be signed by the governing council.
+ * - The newly created node information SHALL be added to the network address
+ *   book information in the network state.
+ * - The new entry SHALL be created in a "pending" state.
+ * - All pending address book entries SHALL be added to the active network
+ *   configuration during the next `freeze` transaction with the field
+ *   `freeze_type` set to `PREPARE_UPGRADE`.
+ *
+ * ### Record Stream Effects
+ * Upon completion the newly assigned `node_id` SHALL be in the transaction
+ * receipt.
  */
 message NodeCreateTransactionBody {
-
     /**
-     * Node account id, mandatory field, ALIAS is not allowed, only ACCOUNT_NUM.
-     * If account_id does not exist, it will reject the transaction.
-     * Multiple nodes can have the same account_id.
+    * A Node account identifier.
+    * <p>
+    * This field is REQUIRED.<br/>
+    * This account identifier MUST be in the "account number" form.<br/>
+    * This account identifier MUST NOT use the alias field.<br/>
+    * If the identified account does not exist, this transaction SHALL fail.<br/>
+    * Multiple nodes MAY share the same node account.
      */
-    AccountID account_id = 1;
+    proto.AccountID account_id = 1;
 
     /**
-     * Description of the node with UTF-8 encoding up to 100 bytes, optional field.
+     * A short description of the node.<br/>
+     * This value, if set, MUST NOT exceed 100 bytes when encoded as UTF-8.<br/>
+     * This value is OPTIONAL.
      */
     string description = 2;
 
     /**
-     * Ip address and port, mandatory field. Fully qualified domain name is
-     * not allowed here. Maximum number of these endpoints is 10.
-     * The first in the list is used as the Internal IP address in config.txt,
-     * the second in the list is used as the External IP address in config.txt,
-     * the rest of IP addresses are ignored for DAB phase 2.
+     * A list of service endpoints.<br/>
+     * These endpoints SHALL represent the published endpoints to which other
+     * consensus nodes may _gossip_ transactions.<br/>
+     * These endpoints MUST specify address and port.<br/>
+     * These endpoints MUST NOT specify a DNS name.<br/>
+     * This list MUST NOT be empty.<br/>
+     * This list MUST NOT contain more than `10` entries.<br/>
+     * The first entry in this list SHALL be the endpoint published to all
+     * consensus nodes.<br/>
+     * All other entries SHALL be reserved for future use.
      */
-    repeated ServiceEndpoint gossip_endpoint = 3;
+    repeated proto.ServiceEndpoint gossip_endpoint = 3;
 
     /**
-     * A node's grpc service IP addresses and ports, IP:Port is mandatory,
-     * fully qualified domain name is optional. Maximum number of these endpoints is 8.
+     * A list of service endpoints.<br/>
```

Comment:
A list of grpc service endpoints. "grpc" needs to be specified in the previous notion discuss.

Reply:
These are service endpoints.  gRPC should be specified in the requirements lines below if that is strictly required (not sure it truly is, but no harm adding it for now).

I'll add the gRPC requirement.